### DISCUSSION TITLE
Hacked

### DISCUSSION THREAD

**Discussion by Niknak1987** [community/community#128479] [Other Feature Feedback, Questions, & Ideas]
What should you do on an android mobile device if you have been hacked, phone fully hacked, emails, WhatsApp, Snapchat etc?

**Comment by DaviZCodes** [comment]
First, identify the source and eliminate it: an antivirus app might help. 

I know many people who have been hacked and the best advice that I gave is to change all your passwords. Change the passwords for your email, WhatsApp, Snapchat, YouTube, games, whatever the hacker might have had access to. After you do this, you should monitor your activity through "Connected devices" or "App Activity," so you can see whether there is still ongoing suspicious activity from the hacker. Set up two factor authentication or security measures to prevent this from occurring again.

This seemed to work for them since they didn't have issues later on.

**Reply by MRSBeauty2u** [reply]
This is me I just found out last year I'm being hacked by someone I've been with fir 30 years and to find out he's still hacking me and full control of all my devices now adding up since October 2021
